docs/source/guide/prompts_overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Prompts overview
+short: Prompts overview
+tier: enterprise
+type: guide
+order: 0
+order_enterprise: 225
+meta_title: Prompts overview
+meta_description: An overview of the Prompts feature in the HumanSignal platform
+section: Prompts
+date: 2024-05-15 14:30:14
+---
+
+
+Use Prompts to evaluate and refine your LLM prompts, and then generate predictions to automate your labeling process. 
+
+All you need to get started is an OpenAI API key and a dataset with some ground truth annotations. 
+
+With Prompts, you can:
+
+* Drastically increase the speed and efficiency of annotations, transforming subject matter experts (SMEs) into highly productive data scientists while reducing the dependency on non-SME manual annotators.
+* Enhance annotation throughput and accuracy, making the process faster and more scalable. 
+* Empower users to harness the full potential of AI-driven text labeling, setting a new standard for efficiency and innovation in data labeling.
+
+## Auto-labeling with Prompts
+ 
+Prompts allows you to leverage LLMs to swiftly generate accurate predictions, enabling instant labeling of thousands of tasks. 
+
+By utilizing AI to handle the bulk of the annotation work, you can significantly enhance the efficiency and speed of your data labeling workflows. This is particularly valuable when dealing with large datasets that require consistent and accurate labeling. Automating this process reduces the reliance on manual annotators, which not only cuts down on labor costs but also minimizes human errors and biases. With AI's ability to learn from the provided ground truth annotations, you can maintain a high level of accuracy and consistency across the dataset, ensuring high-quality labeled data for training machine learning models.
+
+
+
+## Prompt evaluation and fine-tuning
+
+!!! info Tip
+    While you can use Prompts to do basic prompt evaluation and fine-tuning, we recommend using [Evals](evals_overview), which has been specifically designed for this purpose and has much more robust analytics. 
+
+As you evaluate your prompt against the ground truth annotations, we return an accuracy score for each version of your prompt. You can use this to iterate your prompt versions for [clarity, specificity, and context](prompts_draft#Drafting-effective-prompts). 
+
+![Screenshot of accuracy score](/images/prompts/accuracy_score.png)
+
+This accuracy score provides a measurable way to evaluate and refine the performance of your prompt. By tracking accuracy, you can ensure that the automated labels generated by the LLM are consistent with ground truth data. 
+
+This feedback loop allows you to iteratively fine-tune your prompts, optimizing the accuracy of predictions and enhancing the overall reliability of your data annotation processes. In industries where data accuracy directly impacts decision-making and operational efficiency, this capability is invaluable.
+
```

Comment:
Maybe one more section on 'Bootstrapping Projects with Prompts'
This is for the new feature where no GT is needed, and Prompts will not conduct evaluations like accuracy, but the user can generate predictions with a prompt and convert them to annotations.